plugins/PostProcessingPlugin/scripts/MultiExtColorMix.py

```diff
@@ -0,0 +1,623 @@
+            T2_include = False
+        if extruder_count == 3:
+            T2_include = self.getSettingValueByKey("T2_include")
+        enabled_extruders = extruder_count
+        park_head = bool(self.getSettingValueByKey("park_head"))
+        park_x = str(self.getSettingValueByKey("park_x"))
+        park_y = str(self.getSettingValueByKey("park_y"))
+        purge_amt = str(self.getSettingValueByKey("purge_amt"))
+        park_string = ""
+        
+        # If it is a single extruder printer or if only one extruder is enabled then exit with a message-----------
+        if extruder_count == 1 or enabled_extruders < 2:
+            data[0] += ";  Multi-Extruder Color Mixer did not run because - Single extruder printer or single extruder enabled\n"
+            Message(text = "Color Mixer: {}".format("The post processor exited: single extruder printer OR single extruder enabled.")).show()
+            return data
+            
+        # If the printer is not equipped with Shared Heater and Shared Nozzle then exit with a message------------------
+        shared_heater = bool(MyCura.getProperty("machine_extruders_share_heater", "value"))
+        shared_nozzle = bool(MyCura.getProperty("machine_extruders_share_nozzle", "value"))
+        if not shared_heater and not shared_nozzle:
+            Message(text = "Color Mixer: {}".format("This post is for machines with 'Shared Heaters' and 'Shared Nozzles'.  Separate hot ends won't work.  Those settings are NOT enabled in Cura.  The Post Process will exit.")).show()
+            data[0] += ";  Multi-Extruder Color Mixer did not run because - 'Shared Heaters' and/or 'Shared Nozzles' are diabled in Cura.\n"
+            return data
+            
+        # Set variables---------------------------------------------------------------------
+        start_layer = self.getSettingValueByKey("start_layer") - 1
+        end_layer = self.getSettingValueByKey("end_layer")
+        layer_span = end_layer - start_layer
+        resume_extruder = self.getSettingValueByKey("resume_ext")-1
+        if resume_extruder > extruder_count: resume_extruder = extruder_count
+        M164_extruder = extruder_count
+        mix_style = self.getSettingValueByKey("mix_style")
+        
+        # Figure out the actual End Layer-------------------------------------------------------------
+        if end_layer == -1:
+            end_layer = len(data)-3
+        elif end_layer <= start_layer:
+            end_layer = start_layer + 1
+        else:
+            end_layer -= 1            
+        layer_span = end_layer - start_layer
+        
+        # If an extruder is enabled in Cura and included in this mix then calculate the 'Gradient' Indexing Factor------------
+        if T0_include:
+            T0_mix_start = int(self.getSettingValueByKey("T0_mix_start"))
+            T0_mix_end = int(self.getSettingValueByKey("T0_mix_end"))
+            T0_ext_incr = ((T0_mix_start - T0_mix_end) / (layer_span - 1))
```

Comment:
Don't need these outer parenthesis 
```suggestion
            T0_ext_incr = (T0_mix_start - T0_mix_end) / (layer_span - 1)
```

Reply:
Removed.
(I think they were an artifact.)